src/dbt/kipptaf/models/google/forms/base/base_google_forms__form_responses.sql

```diff
@@ -37,8 +37,7 @@ with
             }},
 
             row_number() over (
-                partition by
-                    f.form_id, f.item_id, r.respondent_email, rata.is_null_value
+                partition by f.form_id, r.respondent_email, rata.is_null_value
```

Comment:
@cristinabaldor is the point of this to get the most recent overall form response record? that already exists here: https://github.com/TEAMSchools/teamster/blob/4c1ceb1f6506709ecacec3370c622d855d2c3d37/src/dbt/kipptaf/models/google/forms/staging/stg_google_forms__responses.sql#L10-L13

Reply:
yes it is! let me see if it makes a difference, i feel like something wasn't working which is why we added a separate rn calc

Reply:
that would make more sense, yeah. potential problem is that it's also long by response item, so I'm not entirely sure if that will get you the desired outcome

Reply:
gonna resubmit now using dense_rank, which looks like it's working as we want it to -- i didn't understand Kevin's comment, you want me to rename base_forms to Int_forms?